spacy/vectors.pyx

```diff
@@ -336,8 +336,8 @@ cdef class Vectors:
             best_rows[i:i+batch_size] = xp.argpartition(sims, -n, axis=1)[:,-n:]
             scores[i:i+batch_size] = xp.partition(sims, -n, axis=1)[:,-n:]
 
-            if sort:
-                sorted_index = xp.arange(scores.shape[0])[:,None],xp.argsort(scores[i:i+batch_size], axis=1)[:,::-1]
+            if sort and n > 1:
```

Comment:
Tiny nitpick, but house style is to use `>=` and `<`, to make off-by-one errors more visually obvious.

```suggestion
            if sort and n >= 2:
```